server/controllers/match/placementRequests/occupancyViewController.ts

```diff
@@ -190,28 +203,27 @@ export default class {
     return async (req: ViewDayRequest, res: Response) => {
       const { token } = req.user
       const { id, premisesId, date } = req.params
+      const { criteria = [], excludeSpaceBookingId } = req.query
 
-      const searchState = this.spaceSearchService.getSpaceSearchState(id, req.session)
-
-      if (!searchState) {
-        return res.redirect(paths.v2Match.placementRequests.search.spaces({ id }))
-      }
-
-      const backLink = paths.v2Match.placementRequests.search.occupancy({ id, premisesId })
+      const backLink = req.headers.referer
```

Comment:
@bobmeredith I guess this would probably benefit from the new session-based back link treatment?